src/torch/9-similarity-db/similarity-db.lua

```diff
@@ -38,12 +38,6 @@ function similarityDb()
       similarities[test] = similarity or -1
     end
 
-    -- compare itself with its mirror
-    local flipped = tiefvision_commons.dataPath('encoded-images-flipped', reference)
-    local flippedEncoding = torch.load(flipped):double()
-    local similarity = similarity_lib.similarity(referenceEncoding, flippedEncoding)
-    similarities[reference] = similarity or -1
-
```

Comment:
As explained in the PR description, the similarities generated with the whole images are comparable to those generated with the cropped images. By removing the cropping process, we gain the possibility of running the similarity with any type of images without any manual intervention. 

Flipped images not being used by the unsupervised learning, except for comparing an image to itself, I saw little points in keeping them.